templates/experimental/worker-dart/README.md

```diff
@@ -4,24 +4,33 @@ Your [Dart](https://dart.dev/) code in [index.dart](https://github.com/cloudflar
 
 In addition to [Wrangler](https://github.com/cloudflare/wrangler) you will need to [install Dart](https://dart.dev/get-dart).
 
-#### Wrangler
+## Setup
 
-To generate using [wrangler](https://github.com/cloudflare/wrangler)
+To create a `my-project` directory using this template, run:
 
+```sh
+$ npm init cloudflare my-project worker-dart --no-delegate-c3
+# or
+$ yarn create cloudflare my-project worker-dart --no-delegate-c3
+# or
+$ pnpm create cloudflare my-project worker-dart --no-delegate-c3
 ```
-wrangler generate projectname https://github.com/cloudflare/dart-worker-hello-world
-```
+
+> **Note:** Each command invokes [`create-cloudflare`](https://www.npmjs.com/package/create-cloudflare) for project creation.
```

Comment:
This is not accurate any more, right? Since the `--no-delegate-c3` actually does not invoke create-cloudflare. And similarly throughout these templates.

Reply:
Isn't this only ever going to invoke `create-cloudflare` though, since that's what `npm init` does. It's not in `wrangler init`, so there's nothing it can't "not delegate" to - it's only using C3?

`delegate-c3` is a `wrangler` option and not a `create-cloudflare` option, right?

Reply:
That's correct. I'm wondering if the  best solution for now might be to run `npm create cloudflare@1 [project-name] [template-name]` since that will essentially copy the template and create the directory as expected.

Reply:
LOL! Sorry I only just realised that it was saying `npm init cloudflare` rather than `npx wrangler init`. 

Good catch @Cherry!

Yeah, I guess just reverting these docs to point at create-cloudflare@1 for now is the simplest solution.